src/utils/type-helpers.js

```diff
@@ -1,4 +1,4 @@
-import vcjs from 'vc-js';
+import vcjs from 'vc-js/lib';
```

Comment:
we import like this so that it doesnt use the precompiled dist vcjs version which includes jsonld signatures by default